### ISSUE TITLE
[Feature Request] Moving RMS Filter

### ISSUE THREAD

**Issue by Bartimaeus-** [facontidavide/PlotJuggler#510]
Request for a Moving RMS filter. 

Pretty much the same thing as the existing Moving Average filter but calculating root mean square instead:

![image](https://user-images.githubusercontent.com/2954254/132144731-578cbd34-02ab-4da9-9185-21916c6085f1.png)

This is useful for analyzing power and current, especially for data that changes sign like in motor drives with regen or AC power circuits

**Comment by Bartimaeus-** [comment]
I've created an example csv dataset with Python that has a sine wave and applied moving RMS for window size of 10,25,100. I've also created a custom timeseries Lua script for moving RMS.

Screenshot below shows the original sine wave as well as the moving RMS from the Python script and the Lua script, with the difference in values for the two plotted in the lower chart, which looks small enough to be rounding error:
![image](https://user-images.githubusercontent.com/2954254/132957908-3d8cd099-8bbc-494e-937a-a8077769bc57.png)

This should aid anyone who wants to do moving RMS or for testing implementation of the filter in PlotJuggler.

@facontidavide feel free to bundle that Lua function with PlotJuggler if it's up to snuff and desired

Files are attached:
[PlotJuggler_movingRMS_Test.zip](https://github.com/facontidavide/PlotJuggler/files/7148353/PlotJuggler_movingRMS_Test.zip)
